src/include/sof/debug/telemetry/telemetry.h

```diff
@@ -0,0 +1,85 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2024 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOF_TELEMETRY_H__
+#define __SOF_TELEMETRY_H__
+
+/* Slot in memory window 2 (Debug Window) to be used as telemetry slot */
+#define SOF_DW_TELEMETRY_SLOT 1
```

Comment:
Mem window  slots usage cannot be speeded through the code. 
Slot 0 is used for mtrace by Zephyr:


```
log_backend_adsp_mtrace.c:

static void mtrace_init(void)
{
	if (ADSP_DW->descs[0].type == MTRACE_LOGGING_SLOT_TYPE(MTRACE_CORE)) {
		return;
	}

	ADSP_DW->descs[0].type = MTRACE_LOGGING_SLOT_TYPE(MTRACE_CORE);
}
```

using a magic number... :( 

that will lead to a mess - "which slot is free?? Who knows..."

please add a define for MTRACE and put both defines in adsp_debug_window.h